### ISSUE TITLE
[question] Configuration file

### ISSUE THREAD

**Issue by jagjordi** [librespot-org/librespot#1286]
Is it possible to use a configuration file to pass the options to librespot?
In the old librespot-java it was possible. Also I saw a reference in a 2018 issue to config.toml
Was this option removed?

**Comment by roderickvd** [comment]
No, not possible. Some downstream packages like Raspotify might allow that, I don't know.
